File: financial-domain-starter/src/main/java/com/nixsolutions/financial/security/exception/NoAccessException.java
package com.nixsolutions.financial.security.exception;

public class NoAccessException extends RuntimeException
{
  public NoAccessException()
  {
    this("No acces to the resource");
  }

  public NoAccessException(String s)
  {
    super(s);
  }

  public NoAccessException(String s, Throwable throwable)
  {
    super(s, throwable);
  }

  public NoAccessException(Throwable throwable)
  {
    super(throwable);
  }
}

File: financial-domain-starter/src/main/kotlin/com/nixsolutions/financial/security/exception/NoAccessException.kt
package com.nixsolutions.financial.security.exception

class NoAccessException : RuntimeException {

  @JvmOverloads constructor(s: String = "No acces to the resource") : super(s) {}

  constructor(s: String, throwable: Throwable) : super(s, throwable) {}

  constructor(throwable: Throwable) : super(throwable) {}
}
